addons/website_blog/static/src/js/s_latest_posts_frontend.js

```diff
@@ -16,10 +16,12 @@ publicWidget.registry.js_get_posts = publicWidget.Widget.extend({
      */
     start: function () {
         var self = this;
-        var limit = self.$target.data('postsLimit') || 3;
-        var blogID = self.$target.data('filterByBlogId');
-        var template = self.$target.data('template') || 'website_blog.s_latest_posts_list_template';
-        var loading = self.$target.data('loading');
+        const data = self.$target[0].dataset;
+        const limit = parseInt(data.postsLimit) || 4;
```

Comment:
Can't it be 3 like before so that the compatibility is not broken? And force 4 on the new snippet xml.

Reply:
Sounds good

Reply:
> Can't it be 3 like before so that the compatibility is not broken? And force 4 on the new snippet xml.

 :laughing: ?